cmd/prometheus/config.go

```diff
@@ -334,7 +334,7 @@ func parsePrometheusURL() error {
 	}
 
 	if ok := govalidator.IsURL(cfg.prometheusURL); !ok {
-		return fmt.Errorf("invalid Prometheus URL: %s", cfg.prometheusURL)
+		return fmt.Errorf("invalid Prometheus URL: %s, allowed characters: a-z 0-9 .", cfg.prometheusURL)
```

Comment:
Also, shouldn't it say: `[A-Za-z0-9\.-]` As in even the `-` symbol is supported.